src/spec-node/containerFeatures.ts

```diff
@@ -80,6 +80,9 @@ export async function extendImage(params: DockerResolverParameters, config: Subs
 		if (params.buildxCacheTo) {
 			args.push('--cache-to', params.buildxCacheTo);
 		}
+		if (params.additionalCacheFroms) {
```

Comment:
Skip if `params.buildNoCache` is set.

Reply:
Just noticed `params.additionalCacheFroms` is always set, no need to check for that.

Reply:
✅